src/TaskRunner/Commands/BuildCommands.php

```diff
@@ -233,4 +234,50 @@ protected function getGitCommitHash(): string
     {
         return Robo::getContainer()->get('repository')->getHead()->getCommitHash();
     }
+
+    /**
+     * Build theme assets (Css and Js).
+     *
+     * @param array $options
+     *   Additional options for the command.
+     *
+     * @return \Robo\Collection\CollectionBuilder
+     *   The collection builder.
+     *
+     * @command toolkit:build-assets
+     *
+     * @option default-theme  theme where to build asstes.
+     *
+     * @aliases tba
+     */
+    public function buildAssets(array $options = [
+        'default-theme' => InputOption::VALUE_REQUIRED,
+        'build-npm-packages' => InputOption::VALUE_REQUIRED,
```

Comment:
this should be optional, as we provide the defaults, right, same for build-npm-mode.

simplify the name for `theme`,  `packages` and `mode`. Also update the dockblock